src/redux/rockets/rocketSlice.jsx

```diff
@@ -0,0 +1,50 @@
+import { createSlice, createAsyncThunk } from "@reduxjs/toolkit";
+import axios from "axios";
+
+const initialState = {
+  rockets : [],
+  isLoading : true
+}
+
+const rocketSlice = createSlice({
+  name: "rocket slice",
+  initialState,
+  // extraReducers : (builder) => {
+    
+  // }
```

Comment:
I kindly recommend not leaving commented code on the final project so the code is spotless and more readable